### DISCUSSION TITLE
what kind of projects need to create?

### DISCUSSION THREAD

**Discussion by ruturaj-gh** [TammyAlok2/Hactoberfest2023_ADSC2#2] [General]
same as the title

**Comment by TammyAlok2** [comment]
You can create any type of project as per your tech stack

**Comment by ruturaj-gh** [comment]
I've already created projects in web development; can I contribute them?

**Reply by TammyAlok2** [reply]
Hello @ruturaj-gh You can add your project in our project section first you have to made a folder named (github username) and then after you can submit your project star the repo also

**Comment by D3mOn10** [comment]
How I can take part in the project because I want to be part of the project
